apps/web/src/components/session/columns.tsx

```diff
@@ -0,0 +1,57 @@
+import { Session } from "@cloud/shared";
+import { Link } from "@tanstack/react-router";
+import { ColumnDef } from "@tanstack/react-table";
+import { ArrowRight } from "lucide-react";
+import { Badge } from "../ui/badge";
+
+// This type is used to define the shape of our data.
+// You can use a Zod schema here if you want.
+
+export const columns: ColumnDef<Session>[] = [
+  {
+    accessorKey: "id",
+    header: "ID",
+  },
+  {
+    accessorKey: "status",
+    header: "Status",
+    // TODO: Return aggregated result
```

Comment:
The status is already, there, (that's what `.select((eb) => withStatus(eb))` does), you just need to update the type inside the `ColumnDef` to accurately reflect what the backend route is returning